Fix query/geo_test

jchiu0 opened the pull request:
If I run the tests in "query", they seem to pass. But when I run tests on everything, they tend to fail.

But for now, at least we fix the logic for geo filters.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/dgraph-io/dgraph/303)
<!-- Reviewable:end -->

ashwin95r commented:
Thanks for the change. Just one small comment. Feel free to submit after addressing it. <img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

---

Review status: 0 of 11 files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

*[geo/s2index.go, line 43 at r1](https://reviewable.io:443/reviews/dgraph-io/dgraph/303#-KVECvXNlRAlbUt-KiOv:-KVECvXOV14ZJmoNXPAV:b-dld4fm) ([raw file](https://github.com/dgraph-io/dgraph/blob/37b6ac54ab7f8db12ea67006f731139a721ea58b/geo/s2index.go#L43)):*
> ```Go
> // IndexTokens returns the tokens to be used in a geospatial index for the given geometry. If the
> // geometry is not supported it returns an error.
> func IndexTokens(g *types.Geo) ([]string, error) {
> ```

Since we changed this, some of the tests which use this function need to be changed as well.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/dgraph-io/dgraph/303#-:-KVEC26pr5mcE7f4mv38:bmfaoq2)*
<!-- Sent from Reviewable.io -->

ashwin95r commented:
Reviewed 11 of 11 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/dgraph-io/dgraph/303)*
<!-- Sent from Reviewable.io -->

jchiu0 commented:
Thanks for the quick response!

---

Review status: 11 of 12 files reviewed at latest revision, 1 unresolved discussion.

---

*[geo/s2index.go, line 43 at r1](https://reviewable.io:443/reviews/dgraph-io/dgraph/303#-KVECvXNlRAlbUt-KiOv:-KVEF6-jY73iwueJuVkB:b-tgcixy) ([raw file](https://github.com/dgraph-io/dgraph/blob/37b6ac54ab7f8db12ea67006f731139a721ea58b/geo/s2index.go#L43)):*
<details><summary><i>Previously, ashwin95r (Ashwin Ramesh) wrote…</i></summary>
> Since we changed this, some of the tests which use this function need to be changed as well.
</details>
Great catch! Fixed. "geo" package now passes.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/dgraph-io/dgraph/303#-:-KVEFAUoU-KeqSG0bbzi:b-uq7g8b)*
<!-- Sent from Reviewable.io -->
